c++/src/kj/compat/tls.c++

```diff
@@ -926,9 +926,13 @@ kj::Promise<kj::AuthenticatedStream> TlsContext::wrapServer(kj::AuthenticatedStr
       return KJ_EXCEPTION(DISCONNECTED, "timed out waiting for client during TLS handshake");
     }).exclusiveJoin(kj::mv(promise));
   }
+  auto peerId = kj::str(stream.peerIdentity->toString());
```

Comment:
It looks like `toString()` returns a `kj::String` already, so the `kj::str()` is unnecessary.

Separately, I think we could save at least one allocation on the happy path if we converted this to a coroutine. Then we could let `stream` live on the "stack" (coroutine frame) and only call its `toString()` in the catch block. If you'd rather not convert this to a coroutine just to report this, that's fine with me, though.